SingularityUI/app/components/taskDetail/TaskFileBrowser.jsx

```diff
@@ -64,18 +70,33 @@ function TaskFileBrowser (props) {
     return _.sortBy(props.files, 'isDirectory').reverse();
   }
 
+  function recentlyModifiedTooltip() {
+    return <Tooltip id="tooltip">File is currently being written to</Tooltip>;
+  }
+
   return (
     <div>
       <Breadcrumbs items={pathItems} />
       <UITable
         data={getFiles() || []}
         keyGetter={(file) => file.name}
+        rowClassName={({mtime}) => { return isRecentlyModified(mtime) ? 'bg-info-light' : null; }}
```

Comment:
The app globally refreshes this data every 60 seconds, and when the window regains focus, so this shouldn't get out of sync.